guides/source/asset_pipeline.md

```diff
@@ -1291,7 +1291,7 @@ config.assets.digest = true
 
 # Precompile additional assets (application.js, application.css, and all
 # non-JS/CSS are already added)
-# config.assets.precompile += %w( search.js )
+# config.assets.precompile += %w(admin.js admin.css)
```

Comment:
Can you add the spaces back everywhere? Our style guide uses a space around the array like `%w( admin.js admin.css )`